paco-classifier-service/main.py

```
@@ -0,0 +1,147 @@
+    warmed" state to probe for; the meaningful, cheap thing to confirm
+    instead is that this process is actually serving HTTP (not just that the
+    OS has a listener on the port, which a bare TCP probe can't tell apart
+    from an app that bound the port and then crashed) and that the weight
+    files this process already validated at import time are still there."""
+    if not BACKGROUND_MODEL_PATH.exists() or not STAFFLINES_MODEL_PATH.exists():
+        raise HTTPException(status_code=503, detail="staffline classifier weights missing")
+    return {"status": "ok"}
+
+
+@app.post("/classify")
+def classify(image: UploadFile = File(...)):
```

Ruff (0.16.1)

[warning] Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)